src/client/client.cpp

```diff
@@ -1009,8 +1009,8 @@ void Client::Send(NetworkPacket* pkt)
 		serverCommandFactoryTable[pkt->getCommand()].reliable);
 }
 
-// Will fill up 12 + 12 + 4 + 4 + 4 bytes
-void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *pkt)
+// Will fill up 12 + 12 + 4 + 4 + 4 + 1 + 1 bytes
```

Comment:
```suggestion
// Will fill up 12 + 12 + 4 + 4 + 4 + 1 + 1 + 1 bytes
```

Reply:
Huh. So that was wrong before? Will fix.